corehq/apps/reports/standard/deployments.py

```diff
@@ -92,6 +92,9 @@ def _columns(self):
                              prop_name='reporting_metadata.last_submissions.commcare_version',
                              alt_prop_name='reporting_metadata.last_submission_for_user.commcare_version',
                              sql_col='last_form_app_commcare_version'),
+            DataTablesColumn(_("Number of unsent forms in user's phone"),
+                             help_text=_("""The number of unsent forms corresponding to app in users' phones"""),
```

Comment:
In a precise way, can we communicate that this would be for last built app when not selected?

Reply:
updated text